src/mjolnir/valhalla_fetch_transit.cc

```diff
@@ -1181,4 +1181,4 @@ int main(int argc, char** argv) {
   stitch(pt, all_tiles, dangling_tiles);
 
   return 0;
-}
+}
```

Comment:
you can revert this file by doing `git checkout origin/master -- src/mjolnir/valhalla_fetch_transit.cc`